doc/mgr/dashboard.rst

```diff
@@ -581,7 +581,7 @@ running/deployed::
 
   $ ceph dashboard set-grafana-api-url <grafana-server-url>  # default: ''
 
-The format of url is : `<protocol>:<IP-address>:<port>`
+The format of url is : `<protocol>://<IP-address>:<port>`
 
```

Comment:
Suggest: `The format of the URL`

Reply:
I was thinking of that also. Then I thought that `url` here refers to the parameter named "url" not "URL" as uniform resource locator, in which case the current spelling would make sense.

Reply:
I pondered that myself, but in that case we'd want \`\`grafana-server-url\`\` instead of \`\`url\`\`.  I think that and `the URL` work equally well.  Yes, we docs folks are a pedantic bunch.